terraform/sentinel/README.md

```diff
@@ -0,0 +1,317 @@
+# DNSimple Policy-as-Code with Terraform and Sentinel
+
+Welcome to the DNSimple's Policy-as-Code with Terraform and Sentinel demo. Taking DNS management to a whole new level, this demo unveils the prowess of policy-as-code through the integration of Terraform, Sentinel, and DNSimple's robust DNS and Domain management platform. The fusion of these tools not only streamlines DNS record and Domain management but also enforces policy adherence, ensuring every change is compliant with your organization's standards.
+
+Here's a sneak peek of what you will delve into in this demo:
+
+- Policy-as-Code Paradigm
+Experience the embodiment of policy-as-code, enabling the automation of policy enforcement in real-time whenever DNS changes are proposed.
+
+- Real-Time Policy Enforcement
+With Sentinel’s policy-as-code framework, enforce organizational and regulatory policies instantly, ensuring every DNS change adheres to the set standards before being applied.
+
+- Version-Controlled Policy Management
+Every policy and DNS change is meticulously tracked in your version control system, fostering a culture of accountability and providing a clear audit trail.
+
+- Collaborative Policy Development
+Promote cross-functional collaboration in developing, reviewing, and refining policies, breaking down silos, and fostering a shared responsibility towards policy adherence.
+
+- Educational Journey
+Whether you are new to Terraform, Sentinel, or DNSimple, this demo provides a guided journey, aiding in grasping the fundamentals and illustrating the practical application of policy-as-code in DNS and Domain management.
+
+By the end of this demo, you'll have a solid understanding of how to utilize Terraform, Sentinel, and DNSimple to establish a policy-driven DNS management framework, empowering your organization to democratize DNS management while ensuring policy adherence.
+
+
+## Prerequisites
+
+- Two domains that are registered at DNSimple. Don't have an account? [Sign up for free](https://dnsimple.com/sign_up) today!
+You can also use the [DNSimple Sandbox](https://developer.dnsimple.com/sandbox/) to test this demo out and get a feel for the platform.
+- API access token with write access to the zone. [Create a new token following our support article](https://support.dnsimple.com/articles/api-access-token/). Please take note of the token and your account ID as you will need them later.
+- Terraform CLI installed on your machine. You can download the latest version from the [Terraform website](https://www.terraform.io/downloads.html). Alternatively, you can use the .devcontainer in this repository to get up and running quickly.
+- GitHub account as this demo uses GitHub Actions to automate the DNS change management process. If you would like to set up a GitOps workflow with GitHub Actions. Alternatively, you can only do the demo locally or directly through Terraform Cloud.
```

Comment:
"If you would like to set up a GitOps workflow with GitHub Actions." This is an incomplete sentence, if we start with a conditional "if", we need a "then" statement/action to follow.